_includes/manuals/nightly/3.1.4_browser_compatibility.md

```diff
@@ -3,9 +3,8 @@ Using the most recent version of a major browser is highly recommended, as Forem
 
 The recommended requirements are as follows for major browsers:
 
-* Google Chrome 54 or higher
-* Microsoft Edge
-* Microsoft Internet Explorer 10 or higher
-* Mozilla Firefox 49 or higher
+* Google Chrome latest
+* Microsoft Edge latest
+* Mozilla Firefox latest
```

Comment:
I don't think this is acceptable. Browsers also have LTS versions and we should always require the latest version.

Reply:
I agree it shouldn't be acceptable but this is what pf4 gives us to work with.
The most we can do is to manually set up the current latest LTS release every time we release foreman. It also means that we will need to be careful with cherry-picks that upgrade the pf package.

Reply:
We should take this as feedback to the PF4 team. Just as a context: Firefox on RHEL is an LTS release. That means a team of Red Hat people don't support their own platform.